package.json

```diff
@@ -35,16 +34,15 @@
     "prop-types": "^15.7.2",
     "raf": "^3.4.1",
     "react": "^16.13.1",
-    "react-i18next": "^10.10.0",
     "react-dom": "^16.13.1",
+    "react-i18next": "^10.10.0",
     "react-redux": "^5.0.7",
     "react-test-renderer": "^16.13.1",
     "redux": "3.6.0",
     "redux-mock-store": "1.2.1",
     "redux-thunk": "2.1.0",
     "reselect": "2.5.4",
-    "rimraf": "2.5.4",
-    "semantic-release": "17.0.8"
+    "rimraf": "2.5.4"
```

Comment:
Yep, just a coma that disappear there but for later, i think we can remove rimraf for a `rm -rf`
But don't care for it right now.
If you want, we could see together the deps for this project :)

Reply:
Yeah I go step by step. I want to remove the dust on this repo, but here the focus is on CVE 👍 
Let's see together the best way to revive the repo.